util/util.go

```diff
@@ -72,3 +74,8 @@ func GetCurrentProcessTimes() (utime int64, stime int64, err error) {
 	}
 	return
 }
+
+// SetGoroutineLabels sets profiler labels for identifying goroutines using the pprof package.
+func SetGoroutineLabels(args ...string) {
```

Comment:
This file appears to be unix-only - see its build tags `//go:build !windows`. So Windows build failed.

```
Error: network\wsNetwork.go:1134:7: undefined: util.SetGoroutineLabels
```